### PR TITLE
Add Tizen Dotnet Sap Samples

### PR DESCRIPTION
**Pull request by ghost**
+ Update Dotnet SAP samples info in README
+ Add `.gitignore` file for `Dotnet/`
+ Add Tizen Dotnet SAP samples

### CODE REVIEW

File: Dotnet/SAP/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Tizen .NET Companion App Samples
+Tizen .NET companion app samples using Samsung Accessory Protocol(SAP).
+
+Open one of the following solutions and build it using [Visual Studio](https://www.visualstudio.com/downloads/) with [Visual Studio Tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen).
```
Comment (poiema72) [inline Dotnet/SAP/README.md:4]: Could you plz check whether this link is right? For my understanding, recently document team is migrating documents from developer.tizen.org to docs.tizen.org.

Comment (Juwols) [reply Dotnet/SAP/README.md:4]: I just received a reply from the document team that the URL link will be maintained.

**Review by Juwols — COMMENTED** [review]
LGTM

**Comment by poiema72** [comment]
LGTM

**Review by Juwols — APPROVED** [review]

**Pull request merged**